Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -70,6 +70,20 @@ class DisconnectSetting: WithAccountSetting {
                 self.settings.profile.removeAccount()
                 self.settings.settings = self.settings.generateSettings()
                 self.settings.SELfirefoxAccountDidChange()
+                let loginsController = UIAlertController(
+                    title: NSLocalizedString("Remove Logins?", comment: "Title of the 'Remove firefox logins' alert"),
```

Comment:
"firefox" throughout should be a brand reference. I think we're also using string identifiers now; @thebnich?